mappings/servers/akyto/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "akyto network",
```

Comment:
the ID is not supposed to include spaces

Either `akytonetwork`, or just `akyto`